src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/EndpointContracts/RazorRequestContext.cs

```diff
@@ -2,45 +2,43 @@
 // Licensed under the MIT license. See License.txt in the project root for license information.
 
 using System;
+using System.Diagnostics.CodeAnalysis;
 using Microsoft.CommonLanguageServerProtocol.Framework;
 using Microsoft.Extensions.Logging;
 
-namespace Microsoft.AspNetCore.Razor.LanguageServer.EndpointContracts;
-
-internal readonly struct RazorRequestContext
+namespace Microsoft.AspNetCore.Razor.LanguageServer.EndpointContracts
```

Comment:
Conscious choice to change this? Just curious..

Reply:
I should note: I don't like things that are half/half, and I think we should decide what we're going to do, one way or the other, and then do a mechanical change PR that moves everything at once, and be done with it. After the feature branch merges :)

Reply:
+1

Reply:
I've been trying to do new files as file-namespace because I like that it's less indenting and bracing. I can refrain from further changes if we want to centralize at a later date.

Reply:
This changes from filke scoped to block scoped though... and in the other PR there was one that went the other way. Thats why I wasn't sure.

Reply:
I'm definitely in favor of file-scoped namespaces. If we want to make that a thing, we should all pick a day to make the change across the repo. It's easy to do, but it disrupts existing branches a bit.